lib/constants.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-const VERSION = '1.86.0';
+const VERSION = '1.88.0';
```

Comment:
Oh, sorry for that, then.
Last times, when I was commiting some CSS/JS changes to the repo, there were some issues because of forgetting about version bump (so users were reporting problems because they had old versions of files in cache).
I was trying to remember about that, to not make this mistake again.
So if the way of updating the number changed since then, I will push the change to return to old number, along with other fixes.